### ISSUE TITLE
Some focusable elements don't get visible focus indicator (severity 7)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1628]
Couple of places in the curator dashboard where the focus indicator is insufficiently visible and doesn't match the rest of the app. (Separate issue from #1625)

### View - Edit links on Curation : Items form
<img width="239" alt="Screen Shot 2020-02-06 at 5 14 39 PM" src="https://user-images.githubusercontent.com/6945691/73992504-3b313980-4904-11ea-8223-f0b5e1267425.png">
Artificially set focus on the image, title, and View link. View link is using the hover state (underline) instead of focus state (blue outline).

### View - Edit - Delete links on Curation : Feature page and Curation : About page
<img width="194" alt="Screen Shot 2020-02-06 at 5 28 24 PM" src="https://user-images.githubusercontent.com/6945691/73993108-22298800-4906-11ea-847f-149912660491.png">

Note that the same links on the Curation : Browse page are correct
<img width="223" alt="Screen Shot 2020-02-06 at 5 29 23 PM" src="https://user-images.githubusercontent.com/6945691/73993183-4c7b4580-4906-11ea-915d-d01a48268dce.png">

### **Configuration : Search** form
<img width="157" alt="Screen Shot 2020-02-06 at 5 23 56 PM" src="https://user-images.githubusercontent.com/6945691/73992932-8ac43500-4905-11ea-848a-f8539645f619.png">
First Options link is in focus.

**Comment by mejackreed** [comment]
@ggeisler @jvine can we huddle quickly about the approach I would like to take here? We are using the classes `btn btn-links` on a lot of `<a>` elements throughout spotlight which is suppressing default link behavior. I would like to remove those classes in favor of us just providing the spacing we want. I don't think this will likely affect any end user experience but hopefully will make things more keyboard friendly. Thought we should chat quickly about the implications.
